build.js

```diff
@@ -0,0 +1,57 @@
+// Dependencies
+var GitHub = require("gh.js")
+  , SameTime = require("same-time")
+  , Logger = require("bug-killer")
+  , Fs = require("fs")
+  ;
```

Comment:
@alysonla From [`npm`](https://www.npmjs.com/). To install them, we have two options:

 - create a [`package.json`](https://docs.npmjs.com/files/package.json) file and have there the `dependencies`
 - `npm install` them manually: `npm i gh.js same-time bug-killer`

(`fs` is Node.js built-in module)

I would choose the first one.